grapple/helpers.py

```diff
@@ -69,18 +76,40 @@ def resolve_singular(self, _, info, **kwargs):
             def resolve_plural(self, _, info, **kwargs):
                 qs = cls.objects
                 if issubclass(cls, Page):
-                    qs = cls.objects.live().public().order_by("-first_published_at")
+                    qs = (
+                        cls.objects.live()
+                        .public()
+                        .specific()
```

Comment:
I am curious as to the need for `.specific()` here and below. Would be good to document

Reply:
Now that you mention, I did a quick test locally and indeed we don't need `.specific()` here 🙂 .
I'm going to remove it.

Reply:
I just pushed one commit with the update.